.github/workflows/ci-deploy-env.yml

```diff
@@ -7,17 +7,21 @@ on:
       - 'master'
       - '*deploy/*'
 
+permissions:
+  id-token: write
+  contents: read
+
 jobs:
-  build-test-deploy:
+  build-push:
```

Comment:
why is this renamed to push if it says deploy :thinking:

Reply:
Because this job builds and pushes, and the additional job dispatches to sysops to deploy.

I thought it could make sense to split into two jobs, so the sysops step needs the first one to be successful.